notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        LabMember().insert1([your_name, your_name.split()[0], your_name.split()[1]], skip_duplicates=True)
</pre><p>This won't work for multi part name (e.g. Kyu Hyun Lee - Lee is last name, Kyu Hyun is first name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>This code is a syntactic copy of common_lab.py, but open to suggestions on how to change common_lab.py to make it work! Will make this:</p><pre class="ql-syntax" spellcheck="false">first_name = 'FirstName'
last_name = 'LastName'
your_name = f'{first_name} {last_name}'
LabMember().insert1([your_name, first_name, last_name], skip_duplicates=True)
</pre><p> </p>

Reply:
<p>I've added a LabMember.insert_from_name  function in PR #110 that you can use here in place of <code>LabMember().insert1([your_name, first_name, last_name], skip_duplicates=True)</code></p>

Reply:
<p>Thanks Ryan!</p>